pages/keplr/index.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+  <div class="session-container">
+    <h2 class="session-title">Use Keplr Browser Extension</h2>
+
+    <div v-if="error" class="error-container">
+      <p>There was an error connecting to the Keplr extension:<br /></p>
+      <p class="error">{{ error }}</p>
+    </div>
+
+    <div v-else-if="loading" class="session-main">
```

Comment:
this should use the same loading logic and page structure as the lunie extension...